spec/git_pretty_accept/app_spec.rb

```diff
@@ -185,9 +185,16 @@
     end
 
     And 'both local and remote PR branch have been updated' do
+      # Delay commit of local_pr_message by 1 full second. Git gem
+      # seems to be confused when commits are too close to each other.
+      sleep 1
```

Comment:
I couldn't get Timecop to change the time. I think it might be because the Git gem is just a wrapper on top of Git, and Git is not affected by the stubbing done by Timecop.